src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -45,4 +49,31 @@ describe('Test suite for dictionary concepts components', () => {
     inputField.find('#remove').first().simulate('click');
     expect(props.removeMappingRow).toHaveBeenCalled();
   });
+
+  it('should render when isNew is true and source not CIEL', () => {
+    const newProps = {
+      ...props,
+      isNew: true,
+      source: 'Snomed',
+    };
+
+    wrapper = mount(<Router>
+      <table><tbody><CreateMapping {...newProps} /></tbody></table>
+    </Router>);
+    const inputs = wrapper.find('input');
+    expect(inputs).toHaveLength(3);
```

Comment:
Would this fail if i changed the above source to CIEL?

Reply:
Would this fail if i changed the above isNew to false?

Reply:
Yes. Changing the source to CIEL or isNew to false causes the assertion to fail

Reply:
Would the test fail if i changed the source to ciel?

Reply:
It doesn't fail if source is changed to ciel

Reply:
Does that mean these names are case sensitive?

Reply:
there current implementation is case sensitive, but an issue has been raised to fix that

Reply:
I can't login to JIRA to confirm that. But I'm sure Darius pointed that out. I will create the ticket if it doesn't exist.